pkgs/tools/misc/clipboard-jh/default.nix

```diff
@@ -50,6 +50,6 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/Slackadays/clipboard";
     license = licenses.gpl3Only;
     maintainers = with maintainers; [ dit7ya ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ dit7ya ];
    platforms = platforms.linux;
```

Reply:
Is there any particular reason to why you merged this PR before we could add the changes you suggested to the commit? Because this change has made me somewhat confused.

Reply:
I've opened #230119 for a fix.